doc/api/crypto.md

```diff
@@ -1102,8 +1106,11 @@ changes:
     description: Support for RSASSA-PSS and additional options was added.
 -->
 - `object` {string | Object}
+  - `key` {string | Buffer | TypedArray | DataView}
```

Comment:
This change has to be done in the list below the paragraph. All the options are explained there.